.github/workflows/ci.yaml

```diff
@@ -66,7 +66,7 @@ jobs:
 
       # Remove the step after https://github.com/actions/setup-python/issues/713 is solved
       - name: Fix broken pip installation
-        if: ${{ matrix.target.os == 'macos-12' }}
+        if: ${{ matrix.target.os == 'macos-14' }}
```

Comment:
This step can be removed, because this issue was solved

Reply:
Removed this workaround step which was added as workaround for pip issue.